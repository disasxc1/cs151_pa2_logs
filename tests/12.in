1 0 0
2 1 1
3 2 2
4 3 3
22 1 0
33 2 1
44 3 2
55 4 3
99 5 4
