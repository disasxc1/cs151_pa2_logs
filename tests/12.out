1 2 3 4 22 33 44 55
99

